docs/devel/hello-world-gadget.md

```diff
@@ -454,25 +473,28 @@ Only events generated in containers are now printed, and they include the name o
 generating them.
 
 ```bash
-RUNTIME.CONTAINERNAME        PID             COMM            FILENAME
-...
-mycontainer                  225805          cat             /lib/tls/libm.so.6
-mycontainer                  225805          cat             /lib/x86_64/x86_64/libm.so.6
-mycontainer                  225805          cat             /lib/x86_64/libm.so.6
-mycontainer                  225805          cat             /lib/x86_64/libm.so.6
-mycontainer                  225805          cat             /lib/libm.so.6
-mycontainer                  225805          cat             /lib/libresolv.so.2
-mycontainer                  225805          cat             /lib/libc.so.6
-mycontainer                  225805          cat             /dev/null
+RUNTIME.CONTAINERNAME        PID             COMM            FILENAME                        MNTNS_ID
+...                                                                                          4026532256
```

Comment:
Yes, because we just added the `gadget_mntns_id mntns_id` field to the event (without updating the metadata to use the `ns` template), so the `MNTNS_ID` column appears.

Reply:
I meant the `4026532256` in the line with `...` (placeholder line), not the column 😄

Reply:
Aaaahh, you're right hahahaha. Updated!